packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -31,10 +32,28 @@ class MethodChannelCamera extends CameraPlatform {
   final StreamController<CameraEvent> cameraEventStreamController =
       StreamController<CameraEvent>.broadcast();
 
-  Stream<CameraEvent> _events(int cameraId) =>
+  /// The controller we need to broadcast the different events coming
+  /// from handleMethodCall. Specific to general device events.
```

Comment:
nit: can also be updated to one sentence:

```suggestion
  /// from handleMethodCall, specific to general device events.
```